app/src/main/res/values/untranslatableStrings.xml

```diff
@@ -16,12 +16,12 @@
         Nederlands  - Alert Bouterse, Maikel de Vries, Kenny)&lt;br/&gt;
         Polski      - Anett Sichla, Tomasz Maciejewski&lt;br/&gt;
         Português   - Patrick Montenegro&lt;br/&gt;
-        Ру́сский     - Anton Shestakov, zetx16&lt;br/&gt;
+        Русский     - Anton Shestakov, zetx16&lt;br/&gt;
         Svenska     - Erik, Henrik Gripenberg&lt;br/&gt;
         Türkçe      - Nesim&lt;br/&gt;
         繁体中文     - Kagami&lt;br/&gt;
         简体中文     - Alice Chan, Susanne Schulz&lt;br/&gt;
         日本語       - Naofumi        &lt;br/&gt;
 "
     </string>
-</resources>
\ No newline at end of file
+</resources>
```

Comment:
Edited on Github, which does this. If you are strict about unrelated changes, feel free to pick the important bits by hand and close this PR.